hypershift/baremetal/none.md

```diff
@@ -0,0 +1,360 @@
+export PULL_SECRET_NAME="${HOSTED}-pull-secret"
+export MACHINE_CIDR="10.19.138.0/24"
+export OCP_RELEASE_VERSION="4.9.17"
+export OCP_ARCH="x86_64"
+export BASEDOMAIN="example.com"
+
+export PULL_SECRET_CONTENT=$(cat ~/clusterconfigs/pull-secret.txt)
+export SSH_PUB=$(cat ~/.ssh/id_rsa.pub)
+EOF
+source ./myvars
+~~~
+
+* Create a namespace to host the HostedCluster and secrets
+
+~~~sh
+envsubst <<"EOF" | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: ${CLUSTERS_NAMESPACE}
+EOF
+
+export PS64=$(echo -n ${PULL_SECRET_CONTENT} | base64 -w0)
+envsubst <<"EOF" | oc apply -f -
+apiVersion: v1
+data:
+ .dockerconfigjson: ${PS64}
+kind: Secret
+metadata:
+ name: ${PULL_SECRET_NAME}
+ namespace: ${CLUSTERS_NAMESPACE}
+type: kubernetes.io/dockerconfigjson
+EOF
+ 
+envsubst <<"EOF" | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ${HOSTED}-ssh-key
+  namespace: ${CLUSTERS_NAMESPACE}
+stringData:
+  id_rsa.pub: ${SSH_PUB}
+EOF
+~~~
+
+* Create the `hostedcluster` and the `nodepool`:
+
+~~~sh
+envsubst <<"EOF" | oc apply -f -
+apiVersion: hypershift.openshift.io/v1alpha1
+kind: HostedCluster
+metadata:
+  name: ${HOSTED}
+  namespace: ${CLUSTERS_NAMESPACE}
+spec:
+  release:
+    image: "quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_VERSION}-${OCP_ARCH}"
+  pullSecret:
+    name: ${PULL_SECRET_NAME}
+  sshKey:
+    name: "${HOSTED}-ssh-key"
+  networking:
+    serviceCIDR: "172.31.0.0/16"
+    podCIDR: "10.132.0.0/14"
+    machineCIDR: "${MACHINE_CIDR}"
+  platform:
+    type: None
+  infraID: ${HOSTED}
+  dns:
+    baseDomain: ${BASEDOMAIN}
+  services:
+  - service: APIServer
+    servicePublishingStrategy:
+      nodePort:
+        address: api.${HOSTED}.${BASEDOMAIN}
+      type: NodePort
+  - service: OAuthServer
+    servicePublishingStrategy:
+      nodePort:
+        address: api.${HOSTED}.${BASEDOMAIN}
+      type: NodePort
+  - service: OIDC
+    servicePublishingStrategy:
+      nodePort:
+        address: api.${HOSTED}.${BASEDOMAIN}
+      type: None
+  - service: Konnectivity
+    servicePublishingStrategy:
+      nodePort:
+        address: api.${HOSTED}.${BASEDOMAIN}
+      type: NodePort
+  - service: Ignition
+    servicePublishingStrategy:
+      nodePort:
+        address: api.${HOSTED}.${BASEDOMAIN}
+      type: NodePort
+EOF
+
+envsubst <<"EOF" | oc apply -f -
+apiVersion: hypershift.openshift.io/v1alpha1
+kind: NodePool
+metadata:
+  name: ${HOSTED}-workers
+  namespace: ${CLUSTERS_NAMESPACE}
+spec:
+  clusterName: ${HOSTED}
+  nodeCount: 0
+  management:
+    autoRepair: false
+    upgradeType: Replace
+  platform:
+    type: None
+  release:
+    image: "quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_VERSION}-${OCP_ARCH}"
+EOF
+~~~
+
+After a while, a number of pods will be created in the `${CLUSTERS_NAMESPACE}` namespace. Those pods are the control plane of the hosted cluster.
+
+~~~sh
+oc get pods -n ${CLUSTERS_NAMESPACE}
+<insert here>
```

Comment:
Not sure this is intended. Maybe you wanted to add the list of pods?

Reply:
Fixed, thanks!